build/templates/docker-integration-tests.yml

```diff
@@ -148,6 +148,40 @@ stages:
               envVars: |
                 ServiceBusConnectionString=$(Arcus.Worker.ServiceBus.Topic.ConnectionString)
                 APPLICATIONINSIGHTS_CONNECTION_STRING=InstrumentationKey=$(Arcus.ApplicationInsights.InstrumentationKey)
+          - template: 'run-new-project-from-template.yml'
+            parameters:
+              projectName: 'Arcus.Demo.AzureFunctions.EventHubs.InProcess'
+              projectAlias: 'arcus-az-func-eventhubs'
+              options: '--functions-worker inProcess'
+              ports: '$(Arcus.AzureFunctions.EventHubs.InProcess.Port):$(Arcus.AzureFunctions.EventHubs.InProcess.Port)'
+              script: |
+                Get-Content './Arcus.Demo.AzureFunctions.EventHubs.InProcess/Startup.cs' -Raw |
+                  % { $_ -replace '#error', '#warning' -replace '#if DEBUG', '#if RELEASE' } |
```

Comment:
Just wondering why we're doing this ?

Reply:
Oh, the first replacement is to remove any errors so the project can be build. These error statements are now used to inform the users to setup a correct secret store before trying to run the project.
The second replace changes the DEBUG to RELEASE, so as to make sure that the configuration (and so implicitly environment variables) are made available to the secret store.